nav2_behavior_tree/plugins/action/remove_in_collision_goals_action.cpp

```diff
@@ -55,6 +55,13 @@ void RemoveInCollisionGoals::on_tick()
 BT::NodeStatus RemoveInCollisionGoals::on_completion(
   std::shared_ptr<nav2_msgs::srv::GetCosts::Response> response)
 {
+  if (response->costs.size() != input_goals_.size()) {
```

Comment:
handling of the error introduced below

Reply:
^ `success` added would be good as you mentioned instead :-)